openhtf/plugs/uart.py

```diff
@@ -0,0 +1,295 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
I would consider making the filename match the plug name. If it's SerialPlug, maybe it should be serial.py.